Makefile

```diff
@@ -93,22 +91,8 @@ install-js-dev: node-version-check
 	# Add an additional check against `node_modules`
 	yarn check --verify-tree || yarn install --check-files
 
-install-py-dev: ensure-pinned-pip
-ifeq ($(shell command -v wheel 2> /dev/null),)
-	# The Python version installed via pyenv does not come with wheel pre-installed
-	# Installing wheel will speed up installation of Python dependencies
-	@$(PIP) install wheel
-endif
-	@echo "--> Installing Sentry (for development)"
-ifdef BIG_SUR
-	# grpcio 1.35.0 is very painful to compile on Big Sur, and is not really surfaced in testing anyways.
-	# So we set SYSTEM_VERSION_COMPAT=1 to get pip to download grpcio wheels for older MacOS.
-	SYSTEM_VERSION_COMPAT=1 $(PIP) install 'grpcio==1.35.0'
```

Comment:
Can we keep this? I'd prefer if people didn't go and compile grpcio. Again, it takes forever, and is not applicable to testing.